tests/homework18/homework18.spec.js

```diff
@@ -0,0 +1,207 @@
+import { test, expect } from '@playwright/test';
+
+test.describe("Registration Form Tests", () => {
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("");
+    console.log('Page URL:', page.url());
+    const SignUpButton = page.locator('.hero-descriptor_btn.btn.btn-primary');
+    await SignUpButton.click();
+  })
```

Comment:
variable name should not start with upper letter

Reply:
Corrected